src/common/AbstractObject.cpp

```diff
@@ -716,7 +753,7 @@ unsigned int AbstractObject::getActivityCount() const
 	return activitySet.size();
 }
 
-RESQML2_NS::Activity* AbstractObject::getActivity (const unsigned int & index) const
+RESQML2_NS::Activity* AbstractObject::getActivity (const unsigned int index) const
```

Comment:
no need for the const when removing the reference. Both should be removed or both should be kept.

Reply:
Ok. Done.